server.js

```diff
@@ -4277,6 +4280,9 @@ cache(function(data, match, sendBadge, request) {
     try {
       var data = JSON.parse(buffer);
       var pullrequests = data.size;
+      if (typeof pullrequests === 'undefined') {
```

Comment:
In tests I can see that this code is used to handle case, when repo if not found (please correct me if I'm wrong). In this case API will return 404 status code (e.g. `https://bitbucket.org/api/2.0/repositories/atlassian/python-bitbucket/pullrequests_NOT_FOUND/?limit=0&state=OPEN`). How about checking status code before parsing response body?